### PR TITLE
release 22-10-04

### PR DESCRIPTION
**Pull request by antonyc**

### CODE REVIEW

File: content/changelog/2022-10-04-console-release/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Social authentication with Hasura provider.
```
Comment (danieltprice) [inline content/changelog/2022-10-04-console-release/index.md:7]:
```suggestion
- UI: Added social authentication for Hasura, enabling users to sign in using using social networking services supported by Hasura.
```

File: content/changelog/2022-10-04-console-release/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Control Plane: Improved performance of project creation.
```
Comment (danieltprice) [inline content/changelog/2022-10-04-console-release/index.md:8]:
```suggestion
- Improved Neon project creation performance.
```

File: content/changelog/2022-10-04-console-release/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+### Bug fixes
+
+- UI: Database selector allows to select any of created databases.
```
Comment (danieltprice) [inline content/changelog/2022-10-04-console-release/index.md:12]: Is this the database drop-down on the Dashboard tab, under Connection Details?

File: content/changelog/2022-10-04-console-release/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+- Control Plane: Availability checker waits for existing operations queue for a project to be empty before starting.
+- Control Plane: Selecting operations for redo based on correct status.
```
Comment (danieltprice) [inline content/changelog/2022-10-04-console-release/index.md:14]:
```suggestion
- Control Plane: Operations for redo are now selected based on the correct status. 
```

File: content/changelog/2022-10-04-console-release/index.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- Control Plane: Branch creation endpoint V2 is accessible via oauth token.
```
Comment (danieltprice) [inline content/changelog/2022-10-04-console-release/index.md:15]:
```suggestion
- The V2 branch creation endpoint is now accessible using an OAuth token.
```

Comment (danieltprice) [reply content/changelog/2022-10-04-console-release/index.md:15]: HI Anton, 
The file structure for release notes just changed. This file will have to be moved to the appropriate location. Apologies for the inconvenience. Please see: https://github.com/neondatabase/website/pull/206#.

File: content/changelog/2022-10-04-console-release/index.md (hunk 3 above)
Comment (antonyc) [reply content/changelog/2022-10-04-console-release/index.md:12]: @seymourisdead where [this drop-down](https://github.com/neondatabase/cloud/pull/2417/commits/5a9277a2d1e0a29ef179593200b6bd5ab7f587fd) located?

Comment (seymourisdead) [reply content/changelog/2022-10-04-console-release/index.md:12]: in the Connection details and on the SQL Editor page

File: content/changelog/2022-10-04-console-release/index.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+### What's new
+
+- UI: Added social authentication for Hasura, enabling users to sign in using using social networking services supported by Hasura.
+- Improved Neon project creation performance.
+
+### Bug fixes
+
+- UI: Database selector allows to select any of created databases.
```
Comment (danieltprice) [inline content/changelog/2022-10-04-console-release/index.md:12]:
```suggestion
- UI: The Database drop-down menu that appears in the Connection Details widget on the Dashboard and in the Neon SQL Editor now permits selecting any created database.
```

**Review by danieltprice — DISMISSED** [review]

File: content/changelog/2022-10-04-console-release/index.md (hunk 6 above)
Comment (danieltprice) [reply content/changelog/2022-10-04-console-release/index.md:12]: Important: Please note that the file structure for the release notes changed just after you opened this PR. The contents of the changelog file must be moved to a new file at this path:
https://github.com/neondatabase/website/tree/main/content/release-notes/22-10-04.md

File: content/changelog/2022-10-04-console-release/index.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Added social authentication for Hasura, enabling users to sign in using using social networking services supported by Hasura.
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/changelog/2022-10-04-console-release/index.md:7]: Important: Please note that the file structure for the release notes changed just after you opened this PR. The contents of the changelog file must be moved to a new file at this path:
https://github.com/neondatabase/website/tree/main/content/release-notes/22-10-04.md

File: content/release-notes/2022-10-04.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Added social authentication for Hasura, enabling users to sign in using social networking services supported by Hasura.
+- Improved Neon project creation performance.
```
Comment (danieltprice) [inline content/release-notes/2022-10-04.md:8]: Apologies. I missed this in my earlier review. Does this item have a label like "UI"?

File: content/release-notes/2022-10-04.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+### Bug fixes
+
+- UI: The Database drop-down menu that appears in the Connection Details widget on the Dashboard and in the Neon SQL Editor now permits selecting any created database.
+- Control Plane: The availability checker now waits for the project operations queue to be cleared before starting.
+- Control Plane: Operations for redo are now selected based on the correct status. 
+- The V2 branch creation endpoint is now accessible using an OAuth token.
```
Comment (danieltprice) [inline content/release-notes/2022-10-04.md:15]: Can we add a label for this item?

**Review by danieltprice — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**